protocol/v2/qbft/instance/commit.go

```diff
@@ -60,7 +61,8 @@ func commitQuorumForRoundValue(state *specqbft.State, commitMsgContainer *specqb
 	return state.Share.HasQuorum(len(signers)), msgs, nil
 }
 
-func aggregateCommitMsgs(msgs []*specqbft.SignedMessage) (*specqbft.SignedMessage, error) {
+// AggregateCommitMsgs aggregates commit messages.
+func AggregateCommitMsgs(msgs []*specqbft.SignedMessage) (*specqbft.SignedMessage, error) {
```

Comment:
@alonmuroch Maybe it should be moved to signedMessage in spec, as its not really an commit function and more of signedMessage function?